pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -181,6 +181,10 @@ func SetDefaults_Shoot(obj *Shoot) {
 		p := ShootPurposeEvaluation
 		obj.Spec.Purpose = &p
 	}
+
+	if obj.Spec.Kubernetes.Kubelet.FailSwapOn == nil {
```

Comment:
A small unit test in the `defaults_test.go` so that we don't accidentally break the API in the future would be great.

Reply:
added